maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -393,6 +396,9 @@ class Orchestra(
             onCommandGeneratedOutput(command, defects, imageData)
 
             val word = if (defects.size == 1) "defect" else "defects"
+            val message = "Found ${defects.size} possible $word: ${defects.joinToString { it.reasoning }}"
+            updateMetadata(maestroCommand, metadata.copy(logMessages = metadata.logMessages + message))
```

Comment:
Why is the AI reasoning treated as logMessage when it can be a different field on metadata. Log messages are support to be console logs on js scripts inside maestro.